backend/graminsta/post/views.py

```diff
@@ -198,8 +198,10 @@ def get(request):
         response: json format Posts that should be
             displayed at the given user's timeline
         """
+        context = {"request_user_id": request.user.id}
         posts = get_timeline_posts(request.user)
-        return Response(PostSerializer(posts, many=True).data)
+        return Response(PostSerializer(posts, many=True,
```

Comment:
Please try to follow PEP8 to indent your code.

See also: https://www.python.org/dev/peps/pep-0008/#indentation